app/components/donation/donation-container.js

```diff
@@ -1,11 +1,24 @@
 import Ember from 'ember';
 
 const {
-  Component
+  Component,
+  computed: {
+    and, empty, or
+  }
 } = Ember;
 
 export default Component.extend({
   classNames: ['donation-container'],
   donationAmount: 0,
-  projectTitle: null
+  projectTitle: null,
+  wasNewCard: true,
+
+  isNewCard: empty('card.id'),
```

Comment:
@begedin this one is tricky because it's an `ObjectProxy` which has `content: null` if there is none. This was the least hacky way I could think to deal with this.

Should definitely comment, but would prefer a better fix.

I'm not super satisfied with my implementation over all, but hey: it works.